modules/swagger-parser/src/test/java/io/swagger/parser/NetworkReferenceTest.java

```diff
@@ -62,12 +65,43 @@
 
             issue_411_server        = readFile("src/test/resources/nested-network-references/issue-411-server.yaml");
             issue_411_components    = readFile("src/test/resources/nested-network-references/issue-411-remote2.yaml");
+
+            issue_421_json          = readFile("src/test/resources/petstore.json");
         }
         catch(Exception e) {
             e.printStackTrace();
         }
     }
 
+    @Test
+    public void testIssue136() throws Exception {
+        new Expectations() {{
+            remoteUrl.urlToString("http://petstore.swagger.io/v2/swagger.json", new ArrayList<AuthorizationValue>());
+            result = issue_136_json;
+        }};
+
+        String spec =
+                "swagger: '2.0'\n" +
+                        "info:\n" +
+                        "  title: issue 136\n" +
+                        "paths:\n" +
+                        "  /foo:\n" +
+                        "    get:\n" +
+                        "      parameters: []\n" +
+                        "      responses:\n" +
+                        "        200:\n" +
+                        "          description: 'the pet'\n" +
+                        "          schema:\n" +
+                        "            $ref: 'http://petstore.swagger.io/v2/swagger.json#/definitions/Pet'";
+
+        SwaggerDeserializationResult result = new SwaggerParser().readWithInfo(spec);
+
+        Swagger swagger = result.getSwagger();
+        Property property = swagger.getPath("/foo").getGet().getResponses().get("200").getSchema();
+        assertNotNull(property);
+        assertTrue(property instanceof RefProperty);
+    }
+
```

Comment:
Moved + response mocked to get test to pass

Reply:
I have a question why the need to move the test from one class to another?

Reply:
It was easier to do it this way. All of our classes that we need to attach the fake request already exist in the file that I moved it in to. If I kept them where they were then I would need to add new import and setup lines for mocking the request for just those two tests in two separate files. We already have that set up code in the file that I moved them into.